packages/coin-tester/src/components/coins/index.ts

```diff
@@ -8,6 +8,7 @@ import ETH from './eth';
 import CRO from './cro';
 import MATIC from './matic';
 import LUNA from './luna';
+import sol from './coin-sol';
```

Comment:
```suggestion
import sol from './sol';
```

Should only contain the coin symbol.